scripts/graph-batch-score.coffee

```diff
@@ -0,0 +1,106 @@
+# Description:
+#   Script for batch wise score graph.
+#
+# Configuration:
+#   INFO_SPREADSHEET_URL
+#
+# Commands:
+#   hubot graph score fxx -b    for Bar Graph
+#   hubot graph score fxx -p    for Pie Graph
+#
+# Author:
+#   aman-singh7
+
+util = require('./util')
+
+module.exports = (robot) ->
+
+    scorefield = () ->
+        Field = robot.brain.get("scorefield") or {}
+        robot.brain.set("scorefield",Field)
+        Field
```

Comment:
If this is the same scorefield function, don't write it again. Use the existing one